src/sage/calculus/interpolation.pyx

```diff
@@ -55,39 +55,39 @@ cdef class Spline:
 
     This example is in the GSL documentation::
 
-        sage: v = [(i + sin(i)/2, i+cos(i^2)) for i in range(10)]
-        sage: s = spline(v)
-        sage: show(point(v) + plot(s,0,9, hue=.8))
+        sage: v = [(i + sin(i)/2, i+cos(i^2)) for i in range(10)]                       # optional - sage.symbolic
```

Comment:
These are all floats (passed through to GSL), so you can probably avoid this batch of `# optional` tags with `math.sin` and `math.cos`.

Reply:
Yes, there's a lot of code that does this in Sage; also in sage.plot.

I'll rewrite it. Thanks for taking a look at this in detail!